cmd/syncer/root.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Please move this file into a subpackage (see comment above)

Reply:
done